spark-dependencies/pom.xml

```diff
@@ -509,16 +511,25 @@
 
     <profile>
       <id>spark-1.6</id>
+      <activation>
+        <activeByDefault>true</activeByDefault>
+      </activation>
       <properties>
         <spark.version>1.6.1</spark.version>
         <py4j.version>0.9</py4j.version>
         <akka.group>com.typesafe.akka</akka.group>
         <akka.version>2.3.11</akka.version>
         <protobuf.version>2.5.0</protobuf.version>
       </properties>
+    </profile>
 
-      <dependencies>
-      </dependencies>
+    <profile>
+      <id>spark-2.0</id>
```

Comment:
Zeppelin is not supporting spark-2.0 at the moment. And this profile can make user confuse.
Shell we not have this profile until Zeppelin has spark-2.0 support?